modules/test/playwright/tests/portal-workflow-kaleo-designer-web/notification.spec.ts

```diff
@@ -66,15 +72,20 @@ test.beforeEach(async ({apiHelpers}) => {
 			singleApproverWorkflowDefinition
 		);
 
-	workflowDefinitionId = workflowDefinition.id;
+	workflowDefinitionIds.push(workflowDefinition.id);
 });
 
 test.afterEach(async ({apiHelpers, scriptManagementPage}) => {
-	await apiHelpers.headlessAdminWorkflow.deleteWorkflowDefinition(
-		workflowDefinitionId
-	);
+	for (const workflowDefinitionId of workflowDefinitionIds) {
+		await apiHelpers.headlessAdminWorkflow.deleteWorkflowDefinition(
+			workflowDefinitionId
+		);
 
-	await scriptManagementPage.enableScriptManagementConfiguration();
+		workflowDefinitionIds = [];
+	}
+	if (!scriptManagementEnabled) {
+		await scriptManagementPage.enableScriptManagementConfiguration();
```

Comment:
We need to remember to set the `scriptManagementDisable` to false here. 
To avoid call the `enableScriptManagementConfiguration()` function in next testes